tests/streamlit_ui_tests/demos/vanguard.spec.ts

```diff
@@ -0,0 +1,93 @@
+import { FrameLocator, Page, expect, test } from '@playwright/test';
+import { awaitResponse, clickTab, getMitoFrame } from '../utils';
+
+const createNewColumnWithName = async (page: Page, mito: FrameLocator, name: string) => {
+    await clickTab(page, mito, 'Home');
+    await mito.locator('.mito-toolbar-button', { hasText: 'Insert' }).click();
+    await mito.locator('.endo-column-header-container', { hasText: /new-column/ }).dblclick();
+    await mito.locator('.endo-column-header-container input').fill(name);
+    await mito.locator('.endo-column-header-container input').press('Enter', { delay: 100 });
+    await expect(mito.locator('.endo-column-header-container', { hasText: name })).toBeVisible();
+};
```

Comment:
Don't we already have this utility?

Reply:
Yes but this is based off of main, so it doesn't have the useful utilities yet!